docs/playbooks/metrics/change-fail-rate.md

```diff
@@ -1,26 +1,27 @@
 ---
 layout: docs
 title: Change Fail Rate
+authors: [adixon, bfinster, nnicholson, pgibbs]
 side-navigation: docs-navigation.html
 ---
 
 # {{ page.title }}
 
-The percentage of changes that result in negative customer impact, system degradation, or rollback.
+The percentage of changes that result in negative customer impact, or rollback.
 
-`changeFailRate = impactingChangeCount / changeCount`
+`changeFailRate = failedChangeCount / changeCount`
 
-"Software delivery performance is a combination of three metrics: [lead time](./development-cycle-time.html), [release frequency](./deploy-frequency.html), and [MTTR](./mean-time-to-repair.html). Change fail rate is not included, though it is highly correlated."
+"Software delivery performance is a combination of three metrics: [lead time](./development-cycle-time.html), [release frequency](./release-frequency.html), and [MTTR](./mean-time-to-repair.html). Change fail rate is not included, though it is highly correlated."
```

Comment:
This statement is referenced in multiple locations, we should consider an include for Object Orientation.

Reply:
Was debating that. Didn't want to deal with changing paths before coffee. I'm debating having there at all. It's there for a reason internally but may not matter here. Let's demo and get feedback.